### ISSUE TITLE
Improve README with better usage instructions

### ISSUE THREAD

**Issue by chickendude** [chickendude/GlossikaNativeGLS#6]
Update the README and scripts to explain how they should be used and how to add new courses to it.

**Comment by jubalh** [comment]
I think slowly I'm able to add this info myself. Will prepare an PR once `split_gms.py` is through :)

**Comment by chickendude** [comment]
Do you think the readme is clear enough as it is? Or perhaps i should add some notes about being able to split the GMS C and B files. I should also add a section on running the final create_gls.py script as well.

**Comment by jubalh** [comment]
Yes please do that :) If you like my changes you could just merge then and append the info. If  you don't like them you could discard them and write your own.

**Comment by jubalh** [comment]
Only saw now that https://github.com/chickendude/GlossikaNativeGLS/pull/7 got merged already :)
